### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by 952E-348A** [share121/inter-knot#5053] [其他]
到底如何改这个绳网名称啊？这个名称真的不好啊。

**Comment by 952E-348A** [comment]
求求了。

**Reply by Breeze520** [reply]
你这名字怎么长得跟验证码似的

**Comment by KELERSAMA** [comment]
改了也只能在APP上看到 网页上还是原来的名字

**Comment by KELERSAMA** [comment]
楼主的名字是验证码吧

**Comment by BaiHan2333** [comment]
在github网站上改

**Comment by FeiXiao-zm** [comment]
要在github上改  就是你登陆绳网的那个  有点难找 你研究研究吧

**Comment by xiaqizhengzhao** [comment]
![Snipaste_2024-09-17_15-15-38](https://github.com/user-attachments/assets/ac6f8faa-c502-4679-90e3-e60c7726f5fe)
![Snipaste_2024-09-17_15-15-27](https://github.com/user-attachments/assets/898cab24-7940-4c4a-88b2-abf7878d2115)
![Snipaste_2024-09-17_15-13-05](https://github.com/user-attachments/assets/b2328015-f4d3-48c4-803a-21c38564be2a)
![Snipaste_2024-09-17_15-08-23](https://github.com/user-attachments/assets/813f7c39-ab77-467b-8021-10d2dd53b24c)

**Comment by xiaqizhengzhao** [comment]
教程如图

**Comment by xiaqizhengzhao** [comment]
小三月（March 7th）也发了教程可以看他的

**Comment by 952E-348A** [comment]
整好了，非常感谢。
